_rules/aria-state-or-property-valid-value-6a7281.md

```diff
@@ -167,78 +169,62 @@ For value type `URI`, this rule does not require that the destination URI exists
 
 #### Failed Example 1
 
-`aria-required` property with invalid true/false value
+The `aria-required` [attribute value][] of `undefined` is not valid for a `true/false` [value type][].
 
 ```html
 <div role="textbox" aria-required="undefined" aria-label="A required textbox"></div>
 ```
 
 #### Failed Example 2
 
-`aria-expanded` state with invalid true/false/undefined value
+The `aria-expanded` [attribute value][] of `collapsed` is not valid for a `true/false/undefined` [value type][].
 
 ```html
-<div role="button" aria-expanded="mixed">A button</div>
+<div role="button" aria-expanded="collapsed">A button</div>
 ```
 
 #### Failed Example 3
 
-`aria-pressed` state with invalid tristate value
+The `aria-expanded` [attribute value][] of `horizontal` is not valid for a `tristate` [value type][].
 
 ```html
 <div role="button" aria-pressed="horizontal">An other button</div>
 ```
 
 #### Failed Example 4
 
-`aria-errormessage` property with invalid ID reference value, since space is not allowed in a single ID
+The `aria-rowindex` [attribute value][] of `2.5` is not valid for an `integer` [value type][] because it is a decimal number.
 
 ```html
-<div role="textbox" aria-errormessage="error1 error2" aria-label="A textbox with an error"></div>
+<div role="gridcell" aria-rowindex="2.5">Fred</div>
 ```
 
 #### Failed Example 5
 
-`aria-rowindex` property with invalid integer value
+The `aria-valuemin`, `aria-valuemax` and `aria-valuenow` [attribute values][attribute value] are `strings`. These should all be of the `number` [value type][] instead.
 
 ```html
-<div role="gridcell" aria-rowindex="2.5">Fred</div>
+<div role="spinbutton" aria-valuemin="one" aria-valuemax="three" aria-valuenow="two" aria-label="Choose a value"></div>
 ```
 
 #### Failed Example 6
 
-`aria-valuemin`, `aria-valuemax` and `aria-valuenow` property with invalid number values
+The `aria-live` [attribute value][] of `page` is not a valid `token`, because `page` is a token for `aria-live`.
```

Comment:
```suggestion
The `aria-live` [attribute value][] of `page` is not a valid `token`, because `page` is not a token for `aria-live`.
```
(maybe also add in the list of valid tokens for `aria-page` 🤔 )

Reply:
aria-pabe? This is a token of aria-current, But I think this is enough.